scico/test/linop/test_linop.py

```diff
@@ -609,3 +566,30 @@ def test_slice_blockarray(idx):
     x = BlockArray.array((snp.zeros((3, 4)), snp.ones((3, 4, 5, 6))))
     A = linop.Slice(idx=idx, input_shape=x.shape, input_dtype=x.dtype)
     assert (A @ x).shape == x[idx].shape
+
+
+def test_transpose():
+    shape = (1, 2, 3, 4)
```

Comment:
Why these two new tests, but no corresponding new one for `Sum`?

Reply:
`snp.sum` is tested as part of the reductions tests in `test_numpy`. These two aren't (because they aren't reductions). I can see a case for leaving them out entirely, given that what is really being tested here is the underlying `jax.numpy` function and `jax.linear_transpose` (and all the plumbing that gets those methods called).